apps/omg_child_chain_rpc/lib/omg_child_chain_rpc/application.ex

```diff
@@ -22,7 +22,8 @@ defmodule OMG.ChildChainRPC.Application do
 
   def start(_type, _args) do
     DeferredConfig.populate(:omg_child_chain_rpc)
-
+    DeferredConfig.populate(:omg_watcher_rpc)
```

Comment:
`omg_watcher_rpc` in chch?

and the same vice versa in `omg_watcher_rpc`. If those are intended, a comment would be v. useful

Reply:
yeah, that's kinda the issue we were seeing, tracers are global settings that don't support more then one phoenix application, so watcher's configuration comes in first and :spandex_datadog picks that one.